src/measure_theory/prod_group.lean

```diff
@@ -0,0 +1,190 @@
+/-
+Copyright (c) 2021 Floris van Doorn. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Floris van Doorn
+-/
+import measure_theory.prod
+import measure_theory.group
+
+/-!
+# Measure theory in the product of groups
+
+In this file we show properties about measure theory in products of topological groups
+and properties of iterated integrals in topological groups.
+
+These lemmas show the uniqueness of left invariant measures on locally compact groups, up to
+scaling. In this file we follow the proof and refer to the book *Measure Theory* by Paul Halmos.
+The proof seems to contain an omission in §60 Th. A, see
+`measure_theory.measure_lintegral_div_measure` and
+https://math.stackexchange.com/questions/3974485/does-right-translation-preserve-finiteness-for-a-left-invariant-measure
```

Comment:
Do you think you could sketch the math proof here, in a few lines? This would very helpful to follow the proof below without opening Halmos.

Reply:
Does this look good?